iaso/api/org_unit_change_requests/serializers.py

```diff
@@ -248,22 +260,18 @@ def validate_status(self, value):
             raise serializers.ValidationError(f"Must be `{approved}` or `{rejected}`.")
         return value
 
-    def validate_approved_fields(self, value):
-        try:
-            OrgUnitChangeRequest.clean_approved_fields(value)
-        except ValidationError as e:
-            raise serializers.ValidationError(e.messages[0]) from e
-        return value
+    def validate(self, validated_data):
+        instance = OrgUnitChangeRequest(**validated_data)
+        instance.clean()
```

Comment:
Calling `clean()` from the model here allows to delete `def validate_approved_fields()` and to get rid of redundant code.